arangod/RocksDBEngine/RocksDBUpgrade.cpp

```diff
@@ -119,8 +120,10 @@ void arangodb::rocksdbStartupVersionCheck(ArangodServer& server,
       }
     }
   } else {
-    // new DBs are always created with Big endian data-format
-    endianess = RocksDBEndianness::Big;
+    // new DBs are always created with Big endian data-format, Unless
+    // forced by a command line option:
```

Comment:
This is only true in maintainer mode, isn't it? Should this code path not also get a "#ifdef ARANGODB_ENABLE_MAINTAINER_MODE" flag?